modules/cert-manager-install-cli.adoc

```diff
@@ -35,6 +35,24 @@ spec:
   - "cert-manager-operator"
 ----
 
+.. For {cert-manager-operator} v1.15.0 or later, create a YAML file with the following content:
++
+[source, yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-cert-manager-operator
+  namespace: cert-manager-operator
+spec:
+  targetNamespaces: []
+----
++
+[NOTE]
+====
+Starting from {cert-manager-operator} version 1.15.0, it is recommended to install the operator using `AllNamespaces` OLM `installMode`. Older versions can continue using `SingleNamespace` OLM `installMode`. Support for `SingleNamespace` will be removed in future versions.
```

Comment:
Can you please add `OwnNamespace` along with `SingleNamespace` wherever it is used?